homeassistant/components/niko_home_control/light.py

```diff
@@ -62,6 +68,9 @@ def __init__(self, light, data):
         self._unique_id = f"light-{light.id}"
         self._name = light.name
         self._state = light.is_on
```

Comment:
I think it would make sense to tidy up this code to use attributes in a preliminary PR:
```suggestion
        self._attr_unique_id = f"light-{light.id}"
        self._attr_name = light.name
        self._attr_is_on = light.is_on
```

Then you can drop the three properties (unique_id, name, is_on) that will be automatically handled by the base class.